study-builder/studies/testboston/emails/result_report_created.en.html

```diff
@@ -0,0 +1,10 @@
+#set( $text = {
+  "subject": "Your test results are ready!",
+  "salutation": "Dear {{participant_firstName}} {{participant_lastName}},",
+  "result_ready": "Your COVID-19 test results are ready to be viewed.",
+  "button": "Click here to log into the TestBoston portal and view your results.",
+  "contact": "If you have any questions, please contact us at 617-525-4220, info@testboston.org.",
```

Comment:
I'll recommend putting phone and email into the `result_report_created.tmpl.html`. You can get the HTML/CSS styling for those in other `.tmpl.html` files. If it's too much work, we can come back to it later.